tests/integration/config.go

```diff
@@ -196,7 +236,11 @@ func (s *TestRun) ValidateStringLiterals() {
 			panic("ip suffix 253 is reserved for query node")
 		}
 
-		if ipSuffix < 1 || ipSuffix > 252 {
+		if ipSuffix == 252 {
+			panic("ip suffix 252 is reserved for double signing node")
+		}
+
+		if ipSuffix < 1 || ipSuffix > 251 {
```

Comment:
The nittiest nit of all: `ipSuffix < 1 || 251 < ipSuffix` would be easier to read. Apparently there was a study that found people can read `<` more easily than `>` 🤷